src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -55,33 +77,54 @@
 helps['account show'] = """
 type: command
 short-summary: Get the details of a subscription.
-long-summary: If no subscription is specified, shows the current subscription.
+long-summary: >
+    If no subscription is specified, information for the current subscription is returned.
+examples:
+    - name: >
+        Get the current default subscription returning results as JSON.  Use `--output yaml` for plain text. 
+        You can also get details of the current subscription by using `az account list --query "[?isDefault]"`.
+      text: >
+        az account show --output json
+    - name: Get information about a different subscription returning results as a table.
+      text: >
+        az account show --name "My subscription name" --output table
+    - name: Get information about a different subscription using the subscription ID.
+      text: >
+        az account show --name 00000000-0000-0000-0000-000000000000
+    - name: >
+        Get information about a subscription using `query`.  
+        To see available query options, first run `az account show --output json`.
+      text: >
+        az account show --query name
+        az account show --query user.name
+        az account show --query tenantId
+    - name: Return results in plain text.
+      text: >
+        az account show --query tenantId -o tsv
 """
 
 helps['account set'] = """
 type: command
-short-summary: Set a subscription to be the current active subscription.
-"""
-
-helps['account show'] = """
-type: command
-short-summary: Get the details of a subscription.
-long-summary: >-
-    If the subscription isn't specified, shows the details of the default subscription.
-
-
-    When --sdk-auth is used,
-    the output includes credentials that you must protect. Be sure that you do not include these credentials
-    in your code or check the credentials into your source control. As an alternative, consider using
-    [managed identities](https://aka.ms/azadsp-managed-identities) if available to avoid the need to use credentials.
+short-summary: Change the active subscription.
+examples:
+    - name: Change the active subscription using a subscription name
+      text: >
+        az account set --subscription "My subscription name"
+    - name: Change the active subscription using a subscription ID
+      text: >
+        az account set --subscription 00000000-0000-0000-0000-000000000000
+    - name: Change the active subscription using a variable
+      text: >
+        subscriptionId = "$(az account list --query "[?isDefault].id" -o tsv)"
+        az account set --subscription $subscriptionId
```

Comment:
This is also a general usage and should not be included in command-specific example.

Reply:
I agree here. We should avoid shell specific examples in reference documentation

Reply:
Hmmm, I like this example!  :-)  Let's talk more about `general usage` and `shell specific` in our next docs:eng checkin.

Reply:
Per discussion with @chasewilson , we have agreed to test the following guidelines:

Guidelines:
1.	We can include `--query` and `--output` examples when the usefulness of a command requires it.  (Do not assume that every customer has kept the JSON output default.)
2.	We only provide Bash examples when…
a.	GitHub issues or our internal Power BI report “top error commands” shows that many customers are making the same mistake 

b.	Introductory reference commands.  If someone is new to Azure CLI, what commands will they learn first?
i.	az account
ii.	az group
iii.	az role
iv.	??
We aren’t trying to teach Bash.  We are using a handful of popular commands to show common usage patterns.

What we do for introductory reference commands is not a pattern for Azure service commands (vm, webapp…).